tables/vaccines.csv

```diff
@@ -34,4 +34,5 @@ Ad26.COV2.S + mRNA-1273,Mixed,,,2,32
 Nanocovax,Protein subunit,Nanogen Pharmaceutical Biotechnology JSC,Vietnam,2,33
 BNT162b2 + CoronaVac,Mixed,,,2,34
 mRNA-1273.529,mRNA,Moderna,United States,2,35
-mRNA-1273 or mRNA-1273.529,mRNA,Moderna,United States,2,36
\ No newline at end of file
+mRNA-1273 or mRNA-1273.529,mRNA,Moderna,United States,2,36
+Ad26.COV2.S + BNT162b2,Mixed,,,2,37
```

Comment:
@zachandfox if **Ad26.COV2.S + BNT162b2** is not used, please remove it. The subject_vaccine table I think is presenting the same meaning.

Reply:
I see, sorry for my mistake.